runtime/gc_base/GCExtensions.hpp

```diff
@@ -337,6 +339,8 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 		, minimumFreeSizeForSurvivor(DEFAULT_SURVIVOR_MINIMUM_FREESIZE)
 		, freeSizeThresholdForSurvivor(DEFAULT_SURVIVOR_THRESHOLD)
 		, recycleRemainders(true)
+		, disableScanMountedContinuationObject(true)
+		, enableContinuationMountingMutex(true)
```

Comment:
I believe these are just temporary debug flags that you used during testing. I think you can remove them now, so that affected code is easier to read.